lib/board.ex

```diff
@@ -0,0 +1,38 @@
+defmodule Board do
+  @moduledoc """
+  The module that handles the Board logic for a Tic-Tac-Toe game
+  """
+
+  @spec setup_initial_board(non_neg_integer()) :: map()
+  def setup_initial_board(board_dimensions \\ 3) do
+    Map.new(1..board_dimensions * board_dimensions, fn position -> {position, "#{position}"} end)
```

Comment:
I'm curious about the use of `Map` here instead of a `List`.

Reply:
I was mainly just curious if using a `List` would make formatting the board a bit easier.

Reply:
I think it would be a little bit better/easier for formatting as it would take out the `Map.values()` call, but I think then the place_a_symbol would be a little less efficient as it would use a `List.replace_at` instead. I think they are pretty comparable all in all though (pros and cons for both), so I think its good either way 😄.